packages/webapp/pages/[userId]/index.tsx

```diff
@@ -58,17 +56,13 @@ const ProfilePage = ({
     [checkHasCompleted],
   );
 
-  const { selectedHistoryYear, before, after, yearOptions, fullHistory } =
-    useActivityTimeFilter();
+  const before = startOfTomorrow();
+  const after = subMonths(subDays(before, 2), 5);
```

Comment:
not sure if before it took user's set timezone into account, usually finicky with showing dates